app/labor/article_suggester.rb

```diff
@@ -39,14 +44,20 @@ def other_suggestions(max: 4, ids_to_ignore: [])
   def suggestions_by_tag(max: 4)
     Article.published.tagged_with(cached_tag_list_array, any: true)
       .where.not(user_id: article.user_id)
-      .where("organic_page_views_past_month_count > 5")
+      .where(tag_suggestion_query)
       .order(hotness_score: :desc)
       .offset(rand(0..offset))
       .first(max)
   end
 
   def offset
-    Rails.env.production? ? 200 : 0
+    @total_articles_count > 1000 ? 200 : (@total_articles_count / 10)
+  end
+
+  def tag_suggestion_query
+    # Fore big communities like DEV we can look at organic page views for indicator.
```

Comment:
Perhaps avoid mention of DEV specifically:

```suggestion
    # For communities with a large number of existing articles we can look at organic page views for indicator.
```

Reply:
I thought DEV actually was a useful reference as literally the only community that will fall under this umbrella for a while.

I feel like this implementation is naive enough that it should probably be revisited again as we have more active communities in different situations and configs.